### PR TITLE
Fixing issue #5857 Making Access Form similar to Discount Form

### LINKED ISSUE #5857: Make Access Form similar to Discount Form
The discount form was simplified recently. Please simplify the access form in the same way.

Compare discount form
Screenshot from 2020-12-01 21-33-12

### PR DESCRIPTION
**Pull request by sayak2k1maruti**
Fixes #5857 

#### Short description of what this resolves:
As per instruction in isssue #5857 , I have made the access form just like Discount form

#### Changes proposed in this pull request:
1.Pre-fill "Number of access tickets" with "10"
2.Change "Valid from" to "Access valid from"
3.Change "Expires on" to "Access expires on"
4.Show time in the same line with dates
5.Show timezone of the event in the same way as in the wizard
6.Take out "Show more options" switch and always show below option
7.Take out "Status" and always make the status active

Screenshot from 2020-12-03 19-25-28
Screenshot from 2020-12-03 19-25-16

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/templates/components/forms/events/view/create-access-code.hbs [hunk 1]
```diff
@@ -60,65 +40,40 @@
...
             @id="start_time"
             @value={{this.data.validFromTime}}
             @rangePosition="start" />
         </div>
-        <div class="wide field {{if this.device.isMobile 'sixteen' 'five'}}">
-          <label>{{t 'Expires on'}}</label>
+      </div>
+    </div>
+    <div class="field">
+       <label>{{t 'Access Expires on'}}</label>
```
Comment (daretobedifferent18) [inline app/templates/components/forms/events/view/create-access-code.hbs:61]: Pls fix the indentation and finalize the PR

Comment (sayak2k1maruti) [reply app/templates/components/forms/events/view/create-access-code.hbs:61]: Okay thanks

Comment (sayak2k1maruti) [reply app/templates/components/forms/events/view/create-access-code.hbs]: I'd just fixed it

**Review by daretobedifferent18 — APPROVED** [review]

File: app/templates/components/forms/events/view/create-access-code.hbs (hunk 1 above)
Comment (sayak2k1maruti) [reply app/templates/components/forms/events/view/create-access-code.hbs]: But how to finalize PR?

**Review by daretobedifferent18 — APPROVED** [review]

**Comment by daretobedifferent18** [comment]
@iamareebjamal Pls have a look

**Comment by daretobedifferent18** [comment]
@iamareebjamal This PR is finalized. Pls have a look

**Review by sachinchauhan2889 — COMMENTED** [review]
@sayak2k1maruti  please resolve conflict. I am going to review this PR. we want this PR to get merged by this week.

**Comment by sachinchauhan2889** [comment]
continue in https://github.com/fossasia/open-event-frontend/pull/6196

**Pull request closed without merging**